### ISSUE TITLE
Invalid web font from NPM package [FA PRO]

### ISSUE THREAD

**Issue by Nirva-Software** [FortAwesome/Font-Awesome#14589]
We have an issue getting the font since version 5.6.0 (same bug in 5.6.1, 5.6.3 and 5.7.0)
After npm upgrade on @fortawesome package, the file node_modules/@fortawesome/fontawesome-pro/webfonts/fa-solid-900.svg is truncated. It shows only 190Ko when fa-regular-400.svg is 1696Ko.

Regards,

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Sorry, cannot replicate

package.json
```json
{
  "name": "fa5pro",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "dependencies": {
    "@fortawesome/fontawesome-pro": "^5.7.0"
  }
}
```

```
$ node -v
v11.9.0
$ npm -v
6.5.0
```

```sh
$ du -h node_modules/@fortawesome/fontawesome-pro/webfonts/*.svg
648K	node_modules/@fortawesome/fontawesome-pro/webfonts/fa-brands-400.svg
1.8M	node_modules/@fortawesome/fontawesome-pro/webfonts/fa-light-300.svg
1.7M	node_modules/@fortawesome/fontawesome-pro/webfonts/fa-regular-400.svg
192K	node_modules/@fortawesome/fontawesome-pro/webfonts/fa-solid-900.svg
```

If you are not able to solve your issue, please open this issue in the Font Awesome Pro repository at https://github.com/FortAwesome/Font-Awesome-Pro

If you don't have access to the above repository, please make sure you have linked your Font Awesome Account to your GitHub account at https://fontawesome.com/account/services

**Comment by robmadole** [comment]
@Nirva-Software are you sure that you see this issue with `5.6.1` and `5.6.3`? I cannot reproduce this.

I can confirm that there is an issue with `5.7.0`, though.

**Comment by tagliala** [comment]
Same here, cannot reproduce on 5.6.x

```
$ cat package-lock.json | grep fontawesome-pro
    "@fortawesome/fontawesome-pro": {
      "resolved": "https://npm.fontawesome.com/@fortawesome/fontawesome-pro/-/fontawesome-pro-5.6.0.tgz",
$ du -h node_modules/@fortawesome/fontawesome-pro/webfonts/*.svg
732K	node_modules/@fortawesome/fontawesome-pro/webfonts/fa-brands-400.svg
1.7M	node_modules/@fortawesome/fontawesome-pro/webfonts/fa-light-300.svg
1.5M	node_modules/@fortawesome/fontawesome-pro/webfonts/fa-regular-400.svg
1.2M	node_modules/@fortawesome/fontawesome-pro/webfonts/fa-solid-900.svg

$ cat package-lock.json | grep fontawesome-pro
    "@fortawesome/fontawesome-pro": {
      "resolved": "https://npm.fontawesome.com/@fortawesome/fontawesome-pro/-/fontawesome-pro-5.6.1.tgz",
$ du -h node_modules/@fortawesome/fontawesome-pro/webfonts/*.svg
732K	node_modules/@fortawesome/fontawesome-pro/webfonts/fa-brands-400.svg
1.7M	node_modules/@fortawesome/fontawesome-pro/webfonts/fa-light-300.svg
1.5M	node_modules/@fortawesome/fontawesome-pro/webfonts/fa-regular-400.svg
1.2M	node_modules/@fortawesome/fontawesome-pro/webfonts/fa-solid-900.svg

 $ cat package-lock.json | grep fontawesome-pro
    "@fortawesome/fontawesome-pro": {
      "resolved": "https://npm.fontawesome.com/@fortawesome/fontawesome-pro/-/fontawesome-pro-5.6.3.tgz",
$ du -h node_modules/@fortawesome/fontawesome-pro/webfonts/*.svg
736K	node_modules/@fortawesome/fontawesome-pro/webfonts/fa-brands-400.svg
1.7M	node_modules/@fortawesome/fontawesome-pro/webfonts/fa-light-300.svg
1.5M	node_modules/@fortawesome/fontawesome-pro/webfonts/fa-regular-400.svg
1.2M	node_modules/@fortawesome/fontawesome-pro/webfonts/fa-solid-900.svg
```

**Comment by Nirva-Software** [comment]
Sorry guys, error on my package.json dependencies.
I think that changing the version is enough but when I run the install it actually still resolve to 5.7.0...

**Comment by robmadole** [comment]
@Nirva-Software if you leave the `^` in front of the version spec it will do that.

Can you try `5.7.1-1`? I just released this and it looks correct. I think we just have a corrupted package with `5.7.0`.

**Comment by robmadole** [comment]
@Nirva-Software we've released 5.7.1 to fix this. Let me know if it does.
